### PR TITLE
Updated provisioning graphic and began DevRel 44 console docs updates

### PR DESCRIPTION
**Pull request by celanthe**
Added updated provisioning template graphics to bring console docs in line with devrel 44 release, updated docs to reflect these changes, and more

### CODE REVIEW

**Review by QueenOvDroids — APPROVED** [review]

File: docs/home/console.md [hunk 1]
```diff
@@ -141,7 +141,7 @@ If you’d like to simulate an Android device, click **Simulare Device**. A noti
 
 Capabilities such as Apps, Remote View, and Capture Log may not be available on a simulated device.
 
-:::tip The Simulare feature is only available on accounts created after Friday, September 27, 2019. If you became an Esper client before then and would like access to a simulated device, [contact us](mailto:support@esper.io). :::
+***Tip: The Simulare feature is only available on accounts created after Friday, September 27, 2019. If you became an Esper client before then and would like access to a simulated device, please [contact us](mailto:support@esper.io).***
```
Comment (kirananto) [inline docs/home/console.md:144]: @celanthe You could use the :::tip Feature of vuepress...

Just that the ending tag has to be in new line.. 
For more info: https://v1.vuepress.vuejs.org/guide/markdown.html#custom-containers

:::tip
The Simulare feature is only available on accounts created after Friday, September 27, 2019. If you became an Esper client before then and would like access to a simulated device, please [contact us](mailto:support@esper.io).
:::

File: docs/home/console.md [hunk 2]
```diff
@@ -971,6 +969,7 @@ See [here](./console/policy-management/getgoogleaccount.md) to know how to get t
 **NFC** <br> If you want device users to be able to use near-field communication (NFC) on their devices (on those devices that support NFC) set NFC to 'On'; if you don’t, set it to 'Off.' If any of your applications require NFC, make sure this setting is set to 'On.' <br/>
 **Google PlayStore**<br>If you want device users to have access to your Managed Google Play Store, set the Google Play Store setting to 'On'; if you don’t, set it to 'Off.' You’ll still be able to install any approved Google Play Store apps from Esper onto the device. This only applies to GMS devices, as AOSP devices do not include Google Play Store support.<br/>
 **USB Tethering**<br>USB tethering specifies if the user is able to use their device to share a mobile connection to another device such as a laptop. If you’d like device users to let their devices share a mobile connection with other devices, set the USB tethering to 'On'; if you don’t, set it to 'Off.'<br/>
+***Tip: Please note that disabling USB tethering also disables Wi-Fi tethering.***
```
Comment (kirananto) [inline docs/home/console.md:972]: Same case here. ^^ 

Ignore this comment if it's intentional

File: docs/home/console.md [hunk 3]
```diff
@@ -1031,6 +1030,24 @@ Begin by clicking a radio button to choose either Kiosk Mode or Multi-Applicatio
...
+
+The **'Add Application'** section gives you the ability to load any Enterprise apps and approved Managed Google Play apps onto the device when it is provisioned. These apps must have been previously uploaded to Esper as an Enterprise app through **'Apps'** or previously approved through Managed Google Play.
+
+Approved Managed Google Play apps will only appear if you have enrolled your endpoint in Managed Google Play. 
+These apps must have been approved through Play for Work before they can be selected here. 
+Simply find the Play store application you wish to install in the Get Play Store App, and it will be added to the list of apps. 
+Once an Enterprise app has been uploaded into your Esper Cloud Endpoint, select it from the dropdown menu, 
+then select the version, and click on **'Add'** to add it to your template.
+
+***Tip: Before using a managed Google Play account, you need to set up a Managed Google Play account through Esper.***
```
Comment (kirananto) [inline docs/home/console.md:1043]: Same here... ^^

**Review by kirananto — CHANGES_REQUESTED** [review]

File: docs/home/console.md (hunk 2 above)
Comment (celanthe) [reply docs/home/console.md:972]: This is intentional, yes! :) Cheers!

File: docs/home/console.md (hunk 1 above)
Comment (celanthe) [reply docs/home/console.md:144]: This was an intentional change! :) Cheers!

File: docs/home/console.md [hunk 4]
```diff
@@ -2229,17 +2342,18 @@ There is a "Default" group called where devices are automatically added unless o
...
 
 Enter a unique name in the text box and click **Create**. 
 
 ![Groups](./assets/NewConsole/GroupCreateName.png)
 
-The next step is to add devices to the group from the *Default group *(in this screen called *All devices*). This means you first have to remove any devices you want in the new group from any other group they are in to put them in the Default group. To move devices into your new group from the Default group, click the checkbox to the left of the names of the devices that need to be moved and click **Add Selected**, or for a single device once it is selected click on the + button that appears to the right.
+The next step is to move the devices from another group. Click on the group you wish to move devices from and select the specific devices. Click on **Actions - Move Devices.** A new pop-up window will appear asking you to select the destination group. Once selected, click on **Move Devices.** 
 
 ![Groups](./assets/NewConsole/GroupAddDevices.png)
+![Groups](.assets/NewConsole/movegroupsupdated.png)
```
Comment (kirananto) [inline docs/home/console.md:2356]: It is not
.assets/NewConsole/movegroupsupdated.png

It is
./assets/NewConsole/movegroupsupdated.png

**Comment by kirananto** [comment]
Looks good to be merged... Merging...

**Pull request merged**